.github/workflows/unit-tests.yml

```diff
@@ -114,15 +114,27 @@ jobs:
           path: ${{ github.run_id }}
 
       - name: Run tests
+        # Tests in this stage generate code coverage metrics for the repository
+        # Coverage data is uploaded to Codecov in subsequent stages
         env:
           BIONEMO_DATA_SOURCE: ngc
         run: ./ci/scripts/run_pytest.sh --no-nbval --skip-slow
 
+      - name: Run slow tests
+        if: |
+          github.event_name == 'push' ||
+          (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'INCLUDE_SLOW_TESTS'))
+        env:
+          BIONEMO_DATA_SOURCE: ngc
+        run: pytest -m "slow" sub-packages/
+
       - name: Run notebook tests
-        if: ${{ contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS') }}
+        if: |
+          github.event_name == 'push' ||
```

Comment:
why `push` here?

Reply:
so it will run with push to main

Reply:
on:
  pull_request:
    branches: [main]
  push:
    branches: [main]

Reply:
`github.event_name == 'push' ||  (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS')) `

means to run this always for push events and only when label exists for pull events

Reply:
this is going to be a big burden on our CI; we only have the single worker currently

Reply:
ok, changed to github.event_name == 'merge_group'